scormxblock/scromxblockng.py

```diff
@@ -224,7 +228,15 @@ class ScormXBlock(StudioEditableXBlockMixin, ScorableXBlockMixin, XBlock):
         scope=Scope.settings
     )
 
+    
     editable_fields = ('scorm_pkg', 'ratio', 'open_new_tab', 'display_name', 'due', 'has_score', 'icon_class', 'weight', 'scorm_allow_rescore')
+    
+    iframe_url = String(
```

Comment:
- ? Do we really need this field

Reply:
Yeah, in fact no really need

Reply:
https://github.com/Learningtribes/edx_xblock_scorm/pull/29/commits/ecb1cb34c973d1be44244c64338986238a84a41c